sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -238,6 +237,15 @@
     options.setRelease(propertiesProvider.getProperty("release"));
     options.setDist(propertiesProvider.getProperty("dist"));
     options.setServerName(propertiesProvider.getProperty("servername"));
+
+    final String proxyHost = propertiesProvider.getProperty("proxy.host");
+    final String proxyUser = propertiesProvider.getProperty("proxy.user");
+    final String proxyPass = propertiesProvider.getProperty("proxy.pass");
+    final String proxyPort = propertiesProvider.getProperty("proxy.port");
+
+    if (proxyHost != null && proxyPort != null) {
```

Comment:
is port mandatory? wondering if we can default to 80 for example if not set instead of not setting the proxy at all

Reply:
Good point. I'll default to 80. It will be consistent with 1.7